### PR TITLE
Remove obsolete conditionals

### PR DESCRIPTION
**Pull request by nickpell**
@orangejenny

### CODE REVIEW

File: corehq/apps/users/views/__init__.py [hunk 1]
```diff
@@ -578,7 +578,6 @@ def delete_user_role(request, domain):
 
 
 class UserInvitationView(InvitationView):
```
Comment (czue) [inline corehq/apps/users/views/__init__.py]: looks like these two can also be consolidated now.

File: corehq/apps/users/views/__init__.py [hunk 2]
```diff
@@ -577,18 +582,135 @@ def delete_user_role(request, domain):
...
+                    invitation.save()
+                else:
+                    messages.error(request, _("It looks like you are trying to accept an invitation for "
+                                             "{invited} but you are already a member of {entity} with the "
+                                             "account {current}. Please sign out to accept this invitation "
+                                             "as another user.").format(
+                                                 entity=self.inviting_entity,
+                                                 invited=invitation.email,
+                                                 current=request.couch_user.username,
+                                             ))
```
Comment (dimagimon) [inline corehq/apps/users/views/__init__.py]: E126 continuation line over-indented for hanging indent

File: corehq/apps/users/views/__init__.py [hunk 3]
```diff
@@ -577,18 +582,135 @@ def delete_user_role(request, domain):
...
+                return HttpResponseRedirect(self.redirect_to_on_success)
+
+            if not is_invited_user:
+                messages.error(request, _("The invited user {invited} and your user {current} do not match!").format(
```
Comment (dimagimon) [inline corehq/apps/users/views/__init__.py]: E501 line too long (117 > 115 characters)

**Comment by nickpell** [comment]
since a lot of code was moved around in https://github.com/dimagi/commcare-hq/commit/63078e30147772042a4f70415993b44847a808a1 I'm going to do some light-touch QA on staging once conflicts are resolved.

**Comment by nickpell** [comment]
:white_check_mark:

**Comment by orangejenny** [comment]
just one non-blocking comment

**Pull request merged**